docs/build.md

```diff
@@ -0,0 +1,136 @@
+# vue build
+
+`vue build` command gives you a zero-configuration development setup, install once and build everywhere.
+
+## Features
+
+- **Not a boilerplate**: run a single command to develop your app
+- **Out of the box**: ES2015, single-file component with hot reloading and custom CSS preprocessors
+- **Customizable**: populate a `~/.vue/webpack.config.js` for custom webpack config
+- **Single-file component mode**: just run `vue build Component.vue` and boom!
```

Comment:
currently `test it out in the browser` is what it does, so this will conflict with `distribute component`, maybe only enable this behavior by adding `--component` option?